examples/example_glfw_vulkan/main.cpp

```diff
@@ -63,13 +63,23 @@ static VKAPI_ATTR VkBool32 VKAPI_CALL debug_report(VkDebugReportFlagsEXT flags,
 }
 #endif // IMGUI_VULKAN_DEBUG_REPORT
 
+// Helper variable to check the version of the Vulkan SDK
+constexpr auto desired_version = VK_MAKE_VERSION(1, 3, 216);
+
+#if defined(__APPLE__) && VK_HEADER_VERSION >= desired_version
```

Comment:
This doesn't perform the correct test. The preprocessor doesn't know about constexpr variables and substitutes the not-defined name `desired_version` with an integer zero in the comparison.